src/service/feedback/save.ts

```diff
@@ -0,0 +1,16 @@
+import { IFeedback } from '@/model/feedback';
+
+import { collection, addDoc } from 'firebase/firestore';
+import { firestoreDb } from '../firebaseConfig';
+
+export const saveFeedback = async (feedback: IFeedback) => {
+  try {
+    const docRef = await addDoc(collection(firestoreDb, 'feedbacks'), {
+      ...feedback
+    });
```

Comment:
A good practice is to avoid anyone sending data to Firebase (if exceeding a certain limit it will charge a cost on our account)

In this case, since we don't have any login or way to recognize a user, I suggest

```suggestion
    const docRef = await signInAnonymously(getAuth(firebaseApp)).then(() =>
      addDoc(collection(firestoreDb, 'feedbacks'), {
        ...feedback
      })
    );
```

Reply:
Very nice! I didn't know about this!